src/neo4j/cypher-queries/company.js

```diff
@@ -2,8 +2,8 @@ const getShowQuery = () => `
 	MATCH (company:Company { uuid: $uuid })
 
 	OPTIONAL MATCH (company)<-[:HAS_WRITING_ENTITY]-(:Material)<-[:USES_SOURCE_MATERIAL*0..1]-(material:Material)
-		WHERE NOT (company)<-[:HAS_WRITING_ENTITY]-(:Material)<-[:USES_SOURCE_MATERIAL*0..1]-(:Material)
-			<-[:HAS_SUB_MATERIAL]-(material)
+		WHERE NOT EXISTS((company)<-[:HAS_WRITING_ENTITY]-(:Material)<-[:USES_SOURCE_MATERIAL*0..1]-(:Material)
+			<-[:HAS_SUB_MATERIAL]-(material))
```

Comment:
This message would appear for this section's former state:

`This feature is deprecated and will be removed in future versions.`
``Coercion of list to boolean is deprecated. Please consider using `NOT isEmpty(…)` instead.``